staging/provisioning/windows/windowsnodereset.ps1

```diff
@@ -107,7 +107,6 @@ if ($global:CsiProxyEnabled) {
 Write-Log "Starting kubelet service"
 Start-Service kubelet
 
-Write-Log "Starting kubeproxy service"
-Start-Service kubeproxy
+Write-Log "Do not start kubeproxy service since kubelet will restart kubeproxy"
```

Comment:
maybe this should be a comment over `start-service kubelet`?  It makes sense in context here with the diff. Later it might be a bit of an odd logging statement.

Reply:
I planned to log it in windowsnodereset.log so it will not surprise anyone why not restart kubeproxy when checking the logs. How do you think?